bin/ch-test

```diff
@@ -150,6 +149,158 @@ scope_check () {
     esac
 }
 
+scope_to_digit () {
```

Comment:
Good point. The purpose of this is to compare scopes, e.g., `if [[ img_scope -gt scope ]]; then filter...`. This may not be the best approach. And alternative approach would be to make a filter function like:

```bash
if [[ $CH_TEST_SCOPE == quick ]]; then 
    filter 'scope' full standard
elif [[ $CH_TEST_SCOPE == standard ]]; then
    filter 'scope' fill
[...]
filter () {
   mode=$1; shift
   while [[ $# -gt 0 ]]; do
       case $mode in
       scope)
           # do something
           ;;
       builder-exclude)
           # do something
           ;;
       arch-exclude)
           # do something
           ;;
       esac
       shift
   done
}
```

Reply:
You definitely don't want to pack three functions into one with a "mode" selector. I think your current approach is probably fine; just needs some clarification.